R/plot.R

```diff
@@ -0,0 +1,323 @@
+#' @title Static rasterly graphics
+#' @description Render a \code{rasterly} object. 
+#' This object can involve axes, legend, etc to better convert graphics to data.
+#' @name static
+#' @param rasterlyObj A \code{rasterly} object.
+#' @param xlim The x limits (x1, x2) of the plot.
+#' @param ylim The y limits (x1, x2) of the plot.
+#' @param xlab A y label for the plot.
+#' @param ylab A x label for the plot.
+#' @param main A title for the plot.
+#' @param sub A sub title for the plot.
+#' @param interpolate A logical value indicating whether to linearly interpolate the image
+#' (the alternative is to use nearest-neighbour interpolation, which gives a more blocky result).
+#' @param axes Show axes or not.
+#' @param legend Show legend or not.
+#' @param legendlabel The label of legend.
```

Comment:
@z267xu I would call this `legend_label` to match the syntax of other arguments, and would indicate type, default value, and whether optional. 

```suggestion
#' @param legendlabel Character. The label to apply to the figure legend. Default is `NULL`, which omits the figure legend label.
```

Reply:
I agree. So does the `label_main`